### ISSUE TITLE
Unable to add <signalReference> element

### ISSUE THREAD

**Issue by jiaoL06** [pyoscx/scenariogeneration#174]
Is the <signalReference> element unavailable? When can you add this function？

**Comment by mander76** [comment]
Hmm, haven't used that one. need to check how much work it would be.

**Comment by jiaoL06** [comment]
> Hmm, haven't used that one. need to check how much work it would be.

Thanks for your response. Looking forward to updates.

**Comment by mander76** [comment]
@jakobkaths Have you worked with SignalReferences?

**Comment by jakobkaths** [comment]
No, I don't think we use these actively in our examples and I haven't used them yet. But seems like a simple element with few attributes that should be easy enough to add.

**Comment by HLINDST9** [comment]
@jiaoL06 implemented signalReference in branch signalreference, please confirm that it looks ok
